honeycombio/resource_slack_recipient.go

```diff
@@ -27,7 +29,7 @@ func newSlackRecipient() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				Description:  "The Slack channel or username to send the notification to. Must begin with `#` or `@`.",
```

Comment:
Just noticed this: should probably update this description here, or maybe just remove the "Must begin.." altogether?

Reply:
The related doc in docs/resources/slack_recipient.md should get updated to match